include/slang/symbols/Type.h

```diff
@@ -56,7 +56,7 @@ class Type : public Symbol {
     bitwidth_t getBitWidth() const;
 
     /// Gets $bits of the type. Returns zero if the type does not have a statically known size.
-    bitwidth_t bitstreamWidth() const;
+    std::size_t bitstreamWidth() const;
```

Comment:
I think this is a good change, but can you use "size_t" without the std:: prefix, everywhere in the code?

Reply:
Yes, I will change them.